website/pages/en/sunrise.mdx

```diff
@@ -0,0 +1,117 @@
+
+### Should I use a centralized indexing provider?
+
+If you are building in web2, you should. If you are building in web3, the moment you use a centralized indexing provider, you are giving them control of your dapp and data. The Graph’s decentralized network offers [superior quality of service](https://thegraph.com/blog/qos-the-graph-network/), reliability with unbeatable uptime thanks to node redundancy, as well as significantly [lower costs](/network/benefits/), and you won’t be held hostage at the data layer.
+
+With The Graph Network, your subgraph is public and anyone can query it openly, which increases the usage and network effects of your dapp. With a centralized indexing solution, the subgraph is private to the centralized provider.
+
+Here's a detailed breakdown of the benefits of The Graph over centralized hosting:
+
+Resilience and Redundancy: Decentralized systems are inherently more robust and resilient due to their distributed nature. Data isn't stored on a single server or location. Instead, it's served by hundreds of independent Indexers around the globe. This reduces the risk of data loss or service interruptions if one node fails, leading to exceptional uptime (99.99%).
+
+Quality of Service: In addition to the impressive uptime, The Graph Network features a ~106ms median query speed (latency), and higher query success rates compared to hosted alternatives. Read more in [this blog](https://thegraph.com/blog/qos-the-graph-network/).
+
+Censorship Resistance: Centralized systems can become targets for censorship, either through regulatory pressures or network attacks. In contrast, decentralized systems, due to their dispersed architecture, are much harder to censor, ensuring continuous data availability.
+
+Transparency and Trust: Decentralized systems operate openly, enabling anyone to independently verify the data. This transparency builds trust among network participants, as they can verify the system's integrity without relying on a central authority.
+
+Just as you've chosen your blockchain network for its decentralized nature, security, and transparency, opting for The Graph Network is an extension of those same principles. By aligning your data infrastructure with these values, you ensure a cohesive, resilient, and trust-driven development environment.
+
+### Will my hosted service subgraph be supported by the upgrade Indexer?
+
+Yes, the upgrade Indexer will support all subgraphs published during the upgrade window.
+
+However, some subgraphs may not be eligible for indexing rewards, and as a result, may have difficulty attracting further Indexers. For example, indexing rewards may not be available for subgraphs on certain chains. Members from these blockchain communities are encouraged to integrate their chain through the [Chain Integration Process](/chain-integration-overview/).
+
+## WHat is an upgrade Indexer?
+
+### What is an upgrade Indexer? What does "upgrade Indexer" mean?
```

Comment:
Is the repetition of "What is an upgrade Indexer?" (same as the parent heading) intentional?

Reply:
No, this seems to be a typo - thank you for flaging!